Jellyfin.Plugins.Gotify/Jellyfin.Plugins.Gotify.csproj

```diff
@@ -7,8 +7,6 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <None Remove="thumb.jpg" />
```

Comment:
I think this file was kept just in case we add back the thumbnails to plugins.

Reply:
I was wondering about that, but I've removed it from all the other pr's too. Would you like me to re-add it?